shub/schedule.py

```diff
@@ -69,12 +76,15 @@ def cli(spider, argument, set):
 
 
 def schedule_spider(project, endpoint, apikey, spider, arguments=(),
-                    settings=()):
+                    settings=(), priority=2, units=1, tag=()):
```

Comment:
I'd put the default values into constants, like `DEFAULT_PRIORITY` etc, to define it only once.